packages/react/src/primitives/Breadcrumbs/BreadcrumbLink.tsx

```diff
@@ -0,0 +1,60 @@
+import * as React from 'react';
+import classNames from 'classnames';
+
+import { classNameModifierByFlag } from '../shared/utils';
+import {
+  BreadcrumbsLinkProps,
+  ForwardRefPrimitive,
+  Primitive,
+  BaseBreadcrumbLinkProps,
+} from '../types';
+import { ComponentClassNames } from '../shared/constants';
+import { Link } from '../Link';
+import { BreadcrumbsContext } from './BreadcrumbsContext';
+
+const BreadcrumbLinkPrimitive: Primitive<BreadcrumbsLinkProps, 'span'> = (
+  { className, children, href, ...rest },
+  ref
+) => {
+  const { isCurrent, isDisabled } = React.useContext(BreadcrumbsContext);
+  const componentClasses = classNames(
+    ComponentClassNames.BreadcrumbsLink,
+    classNameModifierByFlag(
+      ComponentClassNames.BreadcrumbsLink,
+      'current',
+      isCurrent
+    ),
+    classNameModifierByFlag(
+      ComponentClassNames.BreadcrumbsLink,
+      'disabled',
+      isDisabled
+    ),
+    className
+  );
+
+  const linkProps = {
+    'aria-current': isCurrent ? 'page' : undefined,
+    'aria-disabled': isDisabled ? 'true' : undefined,
+    as: isCurrent ? 'span' : 'a',
```

Comment:
This results in kind of weird HTML, a span with an href:
```
<span class="amplify-link amplify-breadcrumbs__link amplify-breadcrumbs__link--current" href="/react/components/breadcrumbs" aria-current="page">Breadcrumbs</span>
```
What do you think about outputting a `<Text />` element instead or conditionally adding the href?